src/software/ai/hl/stp/play/BUILD

```diff
@@ -189,7 +189,7 @@ cc_library(
         "//shared:constants",
         "//shared/parameter:cpp_configs",
         "//software/ai/evaluation:enemy_threat",
-        "//software/ai/hl/stp/tactic:kickoff_chip_tactic",
+        "//software/ai/hl/stp/tactic:all_tactics",
```

Comment:
I don't think this dependency is necessary

Reply:
Fixed